src/unix/linux_like/android/mod.rs

```diff
@@ -1338,6 +1338,10 @@ pub const TCGETS: ::c_int = 0x5401;
 pub const TCSETS: ::c_int = 0x5402;
 pub const TCSETSW: ::c_int = 0x5403;
 pub const TCSETSF: ::c_int = 0x5404;
+pub const TCGETS2: ::c_ulong = 0x802c542a;
+pub const TCSETS2: ::c_ulong = 0x402c542b;
+pub const TCSETSW2: ::c_ulong = 0x402c542c;
+pub const TCSETSF2: ::c_ulong = 0x402c542d;
```

Comment:
These should probably be `c_int` for consistency with the other values in this file.

Reply:
Yes, good point. Updated.